test/fail_compilation/ctfe16694.d

```diff
@@ -0,0 +1,11 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ctfe16694.d(11): Error: static variable barptr cannot be read at compile time
```

Comment:
Is it possible to explicitly mention that you cannot take the address of the dllimported symbol?